cmd/utils/flags.go

```diff
@@ -81,1013 +81,1227 @@ func NewApp(gitCommit, usage string) *cli.App {
+		Usage:  "The maximum number of open connections to the database",
+		Value:  30,
+		EnvVar: "KLAYTN_DBSYNCER_DB_MAX_OPEN",
 	}
 	ConnMaxLifeTimeFlag = cli.DurationFlag{
-		Name:  "dbsyncer.db.max.lifetime",
-		Usage: "The maximum amount of time a connection may be reused (default : 1h), ex: 300ms, 2h45m, 60s, ...",
-		Value: 1 * time.Hour,
+		Name:   "dbsyncer.db.max.lifetime",
+		Usage:  "The maximum amount of time a connection may be reused (default : 1h), ex: 300ms, 2h45m, 60s, ...",
+		Value:  1 * time.Hour,
+		EnvVar: "KLAYTN_DBSYNCER_DB_MAX_LIFETIME",
 	}
 	BlockSyncChannelSizeFlag = cli.IntFlag{
-		Name:  "dbsyncer.block.channel.size",
-		Usage: "Block received channel size",
-		Value: 5,
+		Name:   "dbsyncer.block.channel.size",
+		Usage:  "Block received channel size",
+		Value:  5,
+		EnvVar: "KLAYTN_DBSYNCER_BLOCK_CHANNEL_SIZE",
 	}
 	DBSyncerModeFlag = cli.StringFlag{
-		Name:  "dbsyncer.mode",
-		Usage: "The mode of dbsyncer is way which handle block/tx data to insert db (multi, single, context)",
-		Value: "multi",
+		Name:   "dbsyncer.mode",
+		Usage:  "The mode of dbsyncer is way which handle block/tx data to insert db (multi, single, context)",
+		Value:  "multi",
+		EnvVar: "KLAYTN_DBSYNCER_MODE",
 	}
 	GenQueryThreadFlag = cli.IntFlag{
-		Name:  "dbsyncer.genquery.th",
-		Usage: "The amount of thread of generation query in multi mode",
-		Value: 50,
+		Name:   "dbsyncer.genquery.th",
+		Usage:  "The amount of thread of generation query in multi mode",
+		Value:  50,
+		EnvVar: "KLAYTN_DBSYNCER_GENQUERY_TH",
 	}
 	InsertThreadFlag = cli.IntFlag{
-		Name:  "dbsyncer.insert.th",
-		Usage: "The amount of thread of insert operation in multi mode",
-		Value: 30,
+		Name:   "dbsyncer.insert.th",
+		Usage:  "The amount of thread of insert operation in multi mode",
+		Value:  30,
+		EnvVar: "KLAYTN_DBSYNCER_INSERT_TH",
 	}
 	BulkInsertSizeFlag = cli.IntFlag{
-		Name:  "dbsyncer.bulk.size",
-		Usage: "The amount of row for bulk-insert",
-		Value: 200,
+		Name:   "dbsyncer.bulk.size",
+		Usage:  "The amount of row for bulk-insert",
+		Value:  200,
+		EnvVar: "KLAYTN_DBSYNCER_BULK_SIZE",
 	}
 	EventModeFlag = cli.StringFlag{
-		Name:  "dbsyncer.event.mode",
-		Usage: "The way how to sync all block or last block (block, head)",
-		Value: "head",
+		Name:   "dbsyncer.event.mode",
+		Usage:  "The way how to sync all block or last block (block, head)",
+		Value:  "head",
+		EnvVar: "KLAYTN_DBSYNCER_EVENT_MODE",
 	}
 	MaxBlockDiffFlag = cli.Uint64Flag{
-		Name:  "dbsyncer.max.block.diff",
-		Usage: "The maximum difference between current block and event block. 0 means off",
-		Value: 0,
+		Name:   "dbsyncer.max.block.diff",
+		Usage:  "The maximum difference between current block and event block. 0 means off",
+		Value:  0,
+		EnvVar: "KLAYTN_DBSYNCER_MAX_BLOCK_DIFF",
 	}
 	AutoRestartFlag = cli.BoolFlag{
 		Name:  "autorestart.enable",
 		Usage: "Node can restart itself when there is a problem in making consensus",
```

Comment:
Don't we need to have EnvVar for this flag?

Reply:
Yes It's my mistake. Reflected, thanks